austin/simulation.py

```diff
@@ -62,7 +66,9 @@ def run(run_list, injectables=None, warm_start=False):
             models=run_list['models'], resume_after=run_list['resume_after'])
         pipeline.close_pipeline()
         chunk.log_write_hwm()
-
+    
+    if calibration:
+        run_auto_ownwership_calibration(injectables)
```

Comment:
why just auto ownership here and not mode choice?

Reply:
also a typo at L71: "ownwership"

Reply:
calibration functionality removed